src/renderers/webgl/WebGLRenderLists.d.ts

```diff
@@ -22,6 +22,7 @@ export interface RenderItem {
 
 export class WebGLRenderList {
 
+	renderItems: Array<RenderItem>;
```

Comment:
Can we tell TypeScript people that this is temporary somehow?

Reply:
I don't think so. If we don't want people to use this field because we'll plan to remove it then I don't think we should add it into the typescript file. That will make more people aware of it via autocomplete.

If we'd like to further denote it has private or unusable we could prefix with a special character and expose it as something like `_renderItems`?

Reply:
@mrdoob I've removed the variable from the typescript definition file. If there's still a hesitation about exposing renderItems and prefixing something like an underscore isn't an acceptable solution then maybe it's best to remove the tests for `finish()` to get this merged and I'll manually validate the behavior in the upcoming render order PRs.

Let me know what you'd prefer!

Reply:
I vote to remove the tests for finish so `renderItems` does not need to be exposed.

Reply:
Above it was mentioned that [exposing this field for the tests would be worth](https://github.com/mrdoob/three.js/pull/19346#discussion_r424155346) it so I feel like I'm getting conflicting direction and I don't know what to listen to. I made all the requested changes to make it mergeable and I've offered other solutions to address the issues as I understand them but I'm not getting comments on it. At this point I feel like I'm left to keep guessing at changes to make until something is agreeable.

The aversion to making code automatically testable is odd to me. It's unclear how or whether the behavior of `finish` was validated in the first place because it can't be queried from the browser, either. Looking at the [PR that introduced `finish`](https://github.com/mrdoob/three.js/pull/18411#discussion_r377900033) there was a bug that adding an automated test could have caught but was only found by carefully tracing the algorithm. In my opinion that's a problem.

It seems the easiest thing to do for now is just remove it, though. I've adjusted the PR to comment out the tests for `finish` and remove the public `renderItems` field. This way I can at least use the test in the upcoming PR.

If having commented out test is still problematic then please let me know and I'll remove it.